spyder/plugins/debugger/widgets/framesbrowser.py

```diff
@@ -149,13 +155,31 @@ def setup(self):
         self.finder.sig_hide_finder_requested.connect(
             self.sig_hide_finder_requested)
 
+        # Widget empty pane
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "debugger",
+            _("Not debugging right now"),
+            _("Please come back later when we're actually debugging")
```

Comment:
```suggestion
            _("Debugging is not active"),
            _("Start a debugging session with the play/pause button (⏯) to "
              "examine the function calls (or stack frames) that were "
              "executed by Python up to the point where your program is "
              "paused")
```

I suggest to use this text for the message we're going to show here.

@CAM-Gerlach, what do you think?

Reply:
This is what I would suggest, as a modification of Carlos' suggestion

```suggestion
            _("Debugging is not active"),
            _("Start a debugging session with the ⏯ button to step through"
              "your program and view the function calls that are "
              "executed by Python.")
```

Reply:
Revised suggestion after further discussion

```suggestion
            _("Not debugging right now"),
            _("Start a debugging session with the ⏯️ button, allowing you to "
              "through your code step by step and see the functions that "
              "Python has run.")
```